journal-writings/modernized/06-chapter-4.adoc

```diff
@@ -387,6 +387,7 @@ and it was a strengthening opportunity to the honest-hearted.
 About this time I wrote an epistle to Friends in the back settlements of North Carolina,
 as follows: To Friends at their Monthly Meeting at New Garden and Cane creek,
 in North Carolina.
+//After I did this in the original, I checked it here and that last sentence right above I put as a letter head. Should I instead make it like this one, or do you want me to make this one a letterhead too?
```

Comment:
Yeah, I think it's better as a `[.letter-heading]`, agreed. Why don't you make them both like that. And also, can you capitalize "creek" in "Cane creek" while you're at it?